src/model/formulations/GenMorRam2017/startstop.jl

```diff
@@ -0,0 +1,95 @@
+# UnitCommitment.jl: Optimization Package for Security-Constrained Unit Commitment
+# Copyright (C) 2020, UChicago Argonne, LLC. All rights reserved.
+# Released under the modified BSD license. See COPYING.md for more details.
+
+"""
+    _add_startup_shutdown_limit_eqs!(model::JuMP.Model, g::Unit)::Nothing
+
+Startup and shutdown limits from Gentile et al. (2017).
+Eqns. (20), (23a), and (23b) in Knueven et al. (2020).
+
+Creates constraints `eq_startstop_limit`,  `eq_startup_limit`, and `eq_shutdown_limit`
+using variables `Gar1962.StatusVars`, `prod_above` from `Gar1962.ProdVars`, and `reserve`.
+
+Constraints
+---
+* `eq_startstop_limit`
+* `eq_startup_limit`
+* `eq_shutdown_limit`
+"""
+function _add_startup_shutdown_limit_eqs!(
+    model::JuMP.Model,
+    g::Unit,
+    formulation_prod_vars::Gar1962.ProdVars,
+    formulation_status_vars::Gar1962.StatusVars,
+)::Nothing
```

Comment:
1. It looks like we are never including this file, so this function is never used
2. This function has the same signature as the function in file `base/unit.jl`, so Julia would just silently replace the previous definition. To make both functions available, we would need to declare them like this:
```julia
function _add_startup_shutdown_limit_eqs!(
    model::JuMP.Model,
    g::Unit,
    ::Gar1962.ProdVars,
    ::Gar1962.StatusVars,
    ::GenMorRam2017.StartStop,
)::Nothing

end
```